[SQLLINE-11] Added command line options section to manual

snuyanzin opened the pull request:
The PR proposes
1. Command option's section in manual
2. Corrections for jline, sqlline versions
3. Corrections for installations and running after installation

it does not change `manual.txt`, `manual.html` as these files will be generated as a result of commands mentioned at https://github.com/julianhyde/sqlline/commit/cb74e0b05976230a697a1e2d589eac89171f9551

fixes #11 
partially fixes #240

julianhyde commented:
It is true that jars can be found at Maven Central. But please mention where SQLLine's source and releases can be found (https://github.com/julianhyde/sqlline/ and https://julianhyde.github.io/sqlline/). Same goes for jline.

The other changes look good.

snuyanzin commented:
Thank you for your comment.
I added info related to sqlline sources, jline3 sources, releases.
Also I added a new section for prompting which was also missed in docs.

Did not get your thought regarding release's page as I could not find any info about releases at 
https://julianhyde.github.io/sqlline/, only docs.
May be https://github.com/julianhyde/sqlline/releases should be mentioned a release page?

Currently I added this one but I can change if you clarify how to get sqlline releases from https://julianhyde.github.io/sqlline/

julianhyde commented:
Yes, my bad, I should add a link to https://github.com/julianhyde/sqlline/releases from https://julianhyde.github.io/sqlline/.

Releases are important to us Apache folks; if you can get the binaries on Maven central, it's not true open source; you have to be able to get the source code that went into that release.

snuyanzin commented:
Ok I see. Thank you for clarification.

I put https://julianhyde.github.io/sqlline/ as a link for sqlline releases
and link to maven central https://repo1.maven.org/maven2/org/jline/ as a link for jline3 releases according to the quote from https://github.com/jline/jline3/blame/master/README.md#L96
>All the jars and releases are available from Maven Central, so you'll find everything at the following location https://repo1.maven.org/maven2/org/jline/.

julianhyde commented:
Merged as 6705dd6. Fixes #11.
